command/operator_generate_root.go

```diff
@@ -23,18 +23,27 @@ import (
 var _ cli.Command = (*OperatorGenerateRootCommand)(nil)
 var _ cli.CommandAutocomplete = (*OperatorGenerateRootCommand)(nil)
 
+type generateRootKind int
+
+const (
+	generateRootRegular  generateRootKind = 1
```

Comment:
Small nit, but you could use `iota` in here.

Reply:
Done.